app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -201,16 +211,17 @@ export default {
       }
       this.subOrgs = Object.freeze(subOrgs) // Don't add reactivity
 
-      const orgs = stats[kind + 's']
+      const orgs = stats.current[kind + 's']
       if (orgs) {
         orgs[0].subOrgs = this.subOrgs
         this.org = Object.freeze(orgs[0]) // Don't add reactivity
         console.log('   ... got our org', this.org)
       }
     },
 
-    async fetchUsingBackgroundJob ({ kind, orgIdOrSlug, includeSubOrgs, country, startDate, endDate }) {
-      const resp = await createJob(JOB_TYPE, { kind, orgIdOrSlug, includeSubOrgs, country, dateRange: { startDate, endDate } })
+    async fetchUsingBackgroundJob ({ kind, orgIdOrSlug, includeSubOrgs, country, startDate, endDate }, includeOther = false) {
```

Comment:
why not just add a key to object instead of adding param? Object provides that flexility